NEWS.md

```diff
@@ -1,3 +1,18 @@
+# Development version (unreleased)
+
+* `[setup-r]`: on R 3.6.x we now use the P3M snapshot from 2024-06-01 by
```

Comment:
Are you saying that we add the snapshot to the front of the repos list, before `/latest`?

Reply:
No, instead of `/latest`.

Reply:
Oooh but "the default CRAN mirror" is? So there's still source to compile from if the dependency constraints require it?

Reply:
Yes.